### ISSUE TITLE
Command line option removal 

### ISSUE THREAD

**Issue by Bakies** [bridgecrewio/checkov#2773]
I've been making a CI pipeline with checkov, and been very frustrated to find command line options keep disappearing in the **_more than 1000 patch versions_** with nothing mentioning options in the change log. Where did `--output-file-path`, `--config-file` go?? The config file flag is still on the root README. These are breaking changes that should warrant more than a bump to the >1000 patch version number.

**Comment by gruebel** [comment]
Hi @Bakies sorry to hear that, but they are still there
```shell
  --output-file-path OUTPUT_FILE_PATH
                        Name for output file. The first selected output via output flag will be saved to the file (default output is cli)
  --config-file CONFIG_FILE
                        path to the Checkov configuration YAML file
```
when you run `checkov -h` they both show up 🙂

**Comment by Bakies** [comment]
Ah shoot, I think I've figured out, it's my bad. I've run pip install and it's got some unknown version instead of the one I picked.

By bridgecrew.io | version:  
Update available  → 2.0.1037
Run pip3 install -U checkov to update
